iontestdata/good/equivs/nonIVMNoOps.ion

```diff
@@ -0,0 +1,8 @@
+// Unannotated top-level symbols with text equivalent to the IVM but without IVM semantics are no-ops.
+
+embedded_documents::[
+    "",
+    "'$ion_1_0'",
+    "$2",
+    "$ion_symbol_table::{symbols:['''$ion_1_0''']} $10"
```

Comment:
Updated through force push, so I'm not sure if you would've gotten an update email.

Rebased this branch on top of the tip of master; removed the now-redundant `good/symbolWithTab.ion`.